### PR TITLE
[10.0][MIG] barcodes_generator_abstract

### PR DESCRIPTION
**Pull request by lasley**
Migration of `barcodes_generator_abstract` to v10. Was simple enough:

* Rename manifest
* Change openerp references to odoo
* Bump version
* Alphabetize imports

Regarding the coveralls- while it may not be incredibly good practice, I propose we let the other modules cover the testing gap that is introduced here. 

As you can see in #66, the drop is way less significant, and I would guess that adding the partner module probably covers the rest. Just my 2 cents on allowing me less work really 😛

### CODE REVIEW

File: barcodes_generator_abstract/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+{
+    'name': 'Generate Barcodes (Abstract)',
+    'summary': 'Generate Barcodes for Any Models',
+    'version': '10.0.1.0.0',
+    'category': 'Tools',
+    'author':
+        'GRAP,'
+        'La Louve,'
+        'Odoo Community Association (OCA)',
+    'website': 'http://www.odoo-community.org',
```
Comment (lasley) [inline barcodes_generator_abstract/__manifest__.py]: https

File: barcodes_generator_abstract/README.rst [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+to make barcode generation works properly.
+
+``sudo pip install pyBarcode``
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+* Go to Settings / Technical / Sequences & Identifiers / Barcode Nomenclatures
```
Comment (lasley) [inline barcodes_generator_abstract/README.rst:45]: Inventory => [Configuration] Barcode Nomenclatures

**Review by legalsylvain — APPROVED** [review]
:+1: Thanks a lot for porting this module. As this module is an abstract module, useless without partner / product module, I'm agree to let the coverage decrease after this PR, and increase avec partner migration.

regards.

**Comment by lasley** [comment]
@legalsylvain - I'd actually like to make another change here. As I noted in [this comment](https://bitbucket.org/whitie/python-barcode/issues/16/pypi-08-release-request#comment-33978213), the core maintainer of pyBarcodes is non-responsive and unwilling to allow new contributors to the project. I have forked the project [on Github](https://github.com/LasLabs/python-barcode) and will be releasing the completely stable but unrelease 0.8 under `python-barcode` in PyPi. I will do this in the next few days and make the pip change here.

re LasLabs/python-barcode#3 & LasLabs/python-barcode#4

**Comment by lasley** [comment]
Sweet turns out there's a [semi-maintained fork](https://github.com/kxepal/viivakoodi/). The owner is active and responding, so depend switched. I'm going to submit a PR to 9 just so we're synced up, even though the change is transparent.

**Comment by legalsylvain** [comment]
Hi, as said on the other PR #76, agree to switch to the new lib. Could you just add a comment on requirement text (I don't know if it's possible), to say that  viivakoodi is a python library for barcode. (it is not very intuitive) (and maybe a line too, on readme.rst file.)

regards.

**Comment by lasley** [comment]
Agreed on the necessity for comment - added.

**Comment by lasley** [comment]
@tedsalmon please review

**Review by tedsalmon — APPROVED** [review]
👍  LGTM!

**Pull request merged**